### ISSUE TITLE
Groups

### ISSUE THREAD

**Issue by Razi91** [kazupon/vue-validator#59]
Currently I'm doing a 'computed' that crawls through fields. 

I have a form with input groups and it's list in menu on left (fixed positioned). Form is long.
It would be nice if I could define "valid-group" for inputs and then check if all inputs in group are valid (and if it has dirty fields) to mark it.

**Comment by kazupon** [comment]
Thank you for your comment !!

I want to understand more about your above comment. 
I hope that you provide the example codes. :smiley_cat:

**Comment by Razi91** [comment]
Sorry, I wrote it fast and incomprehensibly.

For example, currently `v-validate` creates `vm.validation.[model-name]` object. What I mean is create directive `v-validate-group` (for example: `v-validate-group="groupName") which creates`vm.validationGroups.[groupName]` with listed all validations.

Example:

``` html
<h1 
    v-class="has-error: validationGroup.personalData.dirty && validationGroup.personalData.invalid">
Personal data</h1><!-- this could be also a fixed-placed menu on side of the page-->
<input type="text" 
    v-model="form.personalData.name" 
    v-validate="minLength: 3"
    v-validation-group="personalData"></input>
<input type="text" 
    v-model="form.personalData.pesel" 
    v-validate="required, pesel"
    v-validation-group="personalData"></input>
```

(pesel is a type of ID which I have to validate with date of birth, sex and calculate the check digit)
If any of those 2 fields is invalid, header has class `has-error`.

It would be nice if it would be implemented natively.

**Comment by kazupon** [comment]
Thank you for your details example code.

That's sounds good !!
I'll try to consider this feature at #41.

**Comment by kazupon** [comment]
I implemented in 2.0.0-alpha.1
